scripts/run_backend_tests.py

```diff
@@ -209,6 +209,16 @@ def run(self) -> List[concurrent_task_utils.TaskResult]:
             # see that.
             if 'ev_epollex_linux.cc' in str(e):
                 result = run_shell_cmd(exc_list, env=env)
+            # We exempt this block from coverage since it's just added for
+            # debugging a backend flake.
+            elif 'invalid start byte' in str(e):  # pragma: no cover
+                # We sometimes get a UnicodeDecodeError with an "invalid start
```

Comment:
Could you add a comment mentioning the issue where you're tracking this?

Reply:
done